pkg/scheduler/framework/interface.go

```diff
@@ -183,29 +183,28 @@ func (p PluginToStatus) Merge() *Status {
 		return nil
 	}
 
+	statusPrecedenceMap := map[Code]int{
```

Comment:
move this outside the function. And nit: just `statusPrecedence` is fine

Reply:
Done